app/src/main/java/com/appdev/eateryblueandroid/ui/screens/OnboardingScreen.kt

```diff
@@ -0,0 +1,155 @@
+import com.appdev.eateryblueandroid.ui.components.core.TextStyle
+import com.appdev.eateryblueandroid.ui.components.onboarding.OnboardingViewPager
+import com.appdev.eateryblueandroid.ui.viewmodels.ProfileViewModel
+
+@Composable
+fun OnboardingScreen(
+    profileViewModel: ProfileViewModel
+) {
+    val interactionSource = MutableInteractionSource()
+    var stage: Int by remember {
+        mutableStateOf(0)
+    }
+
+    val goBackToMain = {
+        stage = 0
+    }
+
+    val alpha = animateFloatAsState(
+        targetValue = if (stage == 0 || stage == 2) 1f else 0f
+    )
+
+    Box {
+        Column(
+            modifier = Modifier
+                .fillMaxHeight()
+                .fillMaxWidth()
+                .alpha(alpha.value),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxHeight(.9f)
+                    .fillMaxWidth(),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_eaterylogo_blue),
+                    tint = colorResource(R.color.eateryBlue),
+                    modifier = Modifier
+                        .width(96.dp)
+                        .height(96.dp),
+                    contentDescription = null
+                )
+                Text(
+                    text = "Eatery",
+                    color = colorResource(id = R.color.eateryBlue),
+                    textStyle = TextStyle.SUPER_TITLE
+                )
+
+                // Get Started Button
+                Surface(
+                    shape = RoundedCornerShape(24.dp),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 24.dp, start = 48.dp, end = 48.dp),
+                    elevation = 4.dp
+                ) {
+                    Row(
+                        modifier = Modifier
+                            .background(colorResource(id = R.color.white))
+                            .clickable(
+                                interactionSource = interactionSource,
+                                indication = rememberRipple()
+                            ) {
+                                stage = 1
+
+                            },
+                        horizontalArrangement = Arrangement.Center,
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Text(
+                            textStyle = TextStyle.HEADER_H4,
+                            text = "Get Started",
+                            modifier = Modifier.padding(top = 13.5.dp, bottom = 13.5.dp)
+                        )
+                    }
+                }
+            }
```

Comment:
Oh man, I just swapped to a button and it's like half the lines of code for the same (if not easier) result. I only used this terrible model for rounded buttons because it's what somebody did for LoginButton before I was added to the team... I thought there must be some sort of limitation that prevents you from simply using a button but clearly turns out that wasn't the case!

Reply:
Yeah I'm not sure why 😢, there seems to be a lot of the roundaboutness that could be simplified throughout eatery